src/store/ChartStore.js

```diff
@@ -179,15 +180,16 @@ class ChartStore {
         //     minutes: 30,
         // });
 
-        const holderStyle = stxx.chart.panel.holder.style;
+        this.holderStyle = stxx.chart.panel.holder.style;
+
         stxx.append('deleteHighlighted', this.updateComparisons);
         stxx.addEventListener('layout', () => {
             this.saveLayout();
-            this.setChartPanelTop(holderStyle.top);
+            this.updateChartPanelTop();
         });
         stxx.addEventListener('symbolChange', this.saveLayout.bind(this));
         stxx.addEventListener('drawing', this.saveDrawings.bind(this));
-        // stxx.addEventListener('newChart', () => { });
+        stxx.addEventListener('newChart', this.updateChartPanelTop);
```

Comment:
main fix is here. When the chart is first initialised, the chart title needs to be positioned properly